docs/topics/api/ratings.rst

```diff
@@ -27,13 +27,14 @@ user has already posted a rating for the current version of an add-on.
 .. http:get:: /api/v4/ratings/rating/
 
     :query string addon: The :ref:`add-on <addon-detail>` id, slug, or guid to fetch ratings from. When passed, the ratings shown will always be the latest posted by each user on this particular add-on (which means there should only be one rating per user in the results), unless the ``version`` parameter is also passed.
+    :query string exclude_ratings: Exclude ratings by their ``id``. Multiple ratings can be specified, separated by comma(s).
+    :query string filter: The :ref:`filter(s) <rating-filtering-param>` to apply.
+    :query string score: Only include ratings that have been given a specific ``score``. Multiple scores can be specified, separated by comma(s).
```

Comment:
👍 

I like the comma separated scores

Reply:
Considered it, but how?  `filter` looks like `filter=with_deleted,without_empty_body` but score needs values for which star rating(s) you want to include.  Double `=` like `filter=score=2` (if that's even supported)?  Some other syntax like `:` (that we'd only use here)?

Reply:
Mmm true, that's annoying. I wish I had thought of that when adding `filter`... I would have maybe used a more clever format... Ignore this then.